README.md

```diff
@@ -1,3 +1,81 @@
-# i-slide
+# The `<i-slide>` Web component
 
-Web component to display inline slides
+This repository hosts a Web component that may be used to inline individual HTML/PDF slides of a slide set in an HTML page.
+
+***Warning:*** This component is at early stages of development and only supports a minimal set of features!
+
+
+## Usage
+
+```html
+<!DOCTYPE html>
+<html>
+  <body>
+    <h1>A slideset</h1>
+
+    <p>This is slide 1:</p>
+    <i-slide src="https://example.org/myslides.html#1"></i-slide>
+    <p>Followed by slide 2:</p>
+    <i-slide src="https://example.org/myslides.html#2"></i-slide>
+
+    <p>PDF slides work too:</p>
+    <i-slide src="https://example.org/myslides.pdf#page=1"></i-slide>
+
+    <p>Fallback content can be specified:</p>
+    <i-slide src="https://example.org/notfound#4">[Slide 4 of my slide set]</i-slide>
+
+    <p>Content width can be set (300px by default):</p>
+    <i-slide src="https://example.org/myslides.html#5" width="800"></i-slide>
+    <p>Note: the height is set automatically based on the slide's aspect ratio.</p>
+
+    <!-- Web component must be loaded as a module -->
+    <script src="https://tidoust.github.io/i-slide/i-slide.js" type="module"></script>
+  </body>
+</html>
+```
+
+***Important:*** [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) must be enabled for the slide set, meaning that the HTTP server that hosts the slide set needs to return an `Access-Control-Allow-Origin: *` HTTP header. This may already be the case (for instance, CORS is already enabled for all pages under `*.github.io`).
```

Comment:
```suggestion
***Important:*** If the slide set is served from an origin different from the one where the component is used, [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) must be enabled for the slide set, meaning that the HTTP server that hosts the slide set needs to return an `Access-Control-Allow-Origin: *` HTTP header. This may already be the case (for instance, CORS is already enabled for all pages under `*.github.io`).
```